### DISCUSSION TITLE
Swiper Instance on Angular

### DISCUSSION THREAD

**Discussion by MarioC3** [nolimits4web/swiper#4416] [Q&A] [answered]
Hello Guys,

I was wondering how can I get the Swiper Instance on Angular, so I can have access to the methods.

**Comment by vltansky** [comment, marked as answer]
yeah ofcourse!

```html
	<swiper #sliderRef>
```

```js
  @ViewChild("sliderRef", { static: false }) swiperRef?: SwiperComponent;
....
    this.sliderRef.swiperRef.slideNext();
```
see here:
https://stackblitz.com/edit/swiper-angular-example?file=src/app/app.component.ts

**Reply by MarioC3** [reply]
Thank you for this!
